### ISSUE TITLE
[doc] Explain how to create a custom input

### ISSUE THREAD

**Issue by pedrokiefer** [piceaTech/ember-rapid-forms#31]

**Comment by GCorbel** [comment]
@pedrokiefer , you mean something like : 

    {{#em-form model=someModel}}
        {{input value="test"}}
    {{/em-form}}

Is it useful?

**Comment by pedrokiefer** [comment]
Not really, I mean actually extending em-input so you can add say a custom input formatter, or something like that.
